composer.json

```diff
@@ -2,7 +2,6 @@
     "name": "klaviyo/magento2-extension",
     "description": "Klaviyo extension for Magento 2. Allows pushing newsletters to Klaviyo's platform and more.",
     "type": "magento2-module",
-    "version": "1.0.3",
```

Comment:
Totally with you that we don't need this for the case that we're dumping this on packagist and distributing through composer but I think we may still need this for submitting the extension to the Magento Marketplace. Could you add it back and also increment the minor version?

Reply:
Updated version.